rslib/src/search/sqlwriter.rs

```diff
@@ -202,17 +215,34 @@ impl SqlWriter<'_> {
         self.args.push(text);
         let arg_idx = self.args.len();
 
-        let sfld_expr = if no_combining {
-            "coalesce(without_combining(cast(n.sfld as text)), n.sfld)"
+        let process_text_opt = (no_combining as u8) | ((strip_clozes as u8) << 1);
```

Comment:
How about we pull in https://docs.rs/bitflags/latest/bitflags/ and use that instead of magic numbers?